code/modules/client/preference_setup/loadout/loadout.dm

```diff
@@ -422,4 +427,25 @@ var/list/gear_datums = list()
 	else if(H.put_in_hands(item))
 		to_chat(H, "<span class='notice'>Placing \the [item] in your hands!</span>")
 	else
-		to_chat(H, "<span class='danger'>Dropping \the [item] on the ground!</span>")
\ No newline at end of file
+		to_chat(H, "<span class='danger'>Dropping \the [item] on the ground!</span>")
+
+/datum/gear/proc/implant_into_mob(var/mob/living/carbon/human/H, obj/item/I)
+	var/obj/item/organ/external/organ_to_implant_into = H.get_organ(BP_CHEST)
+
+	if(istype(I, /obj/item/organ/internal/augment))
+		var/obj/item/organ/internal/augment/A = I
+		var/implantloc = A.parent_organ
+		organ_to_implant_into = H.get_organ(implantloc)
+		if(A.augment_flags == AUGMENTATION_MECHANIC)
+			if(!BP_IS_PROSTHETIC(organ_to_implant_into))
+				to_chat(H, "<span class='warning'>Your [organ_to_implant_into.name] is not prosthetic, and therefore the [A] can not be installed!</span>")
```

Comment:
SPAN_WARNING (or maybe SPAN_DANGER since it's a serious failure, not a minor one).